ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -133,7 +133,8 @@ def test_jax_silu(
 @handle_frontend_test(
     fn_tree="jax.nn.leaky_relu",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float_and_integer"),
+        available_dtypes=helpers.get_dtypes("float_and_integer")
```

Comment:
Can't you just do `helpers.get_dtypes("numeric")`?
that contains complex dtypes as well.

Reply:
That's a good point, updating it and the other tests now. Thanks for pointing it out